Dockerfile.dapper

```diff
@@ -22,7 +22,15 @@ ENV K3S_BINARY_amd64=k3s \
     K3S_BINARY_s390x=k3s-s390x \
     K3S_BINARY=K3S_BINARY_${ARCH}
 
-RUN curl -sL https://github.com/rancher/k3s/releases/download/$(curl -Ls -o /dev/null -w %{url_effective} https://update.k3s.io/v1-release/channels/v1.24 | awk -F/ '{ print $NF }')/${!K3S_BINARY} > /usr/local/bin/k3s && \
+ARG K8S_VERSION_FROM_DRONE
+ENV K8S_VERSION $K8S_VERSION_FROM_DRONE
+
+RUN if [ -z "${K8S_VERSION}" ]; then export K8S_VERSION="v1.24" && echo $(date +%s%N); fi
+# ENV K8S_VERSION="${K8S_VERSION_FROM_DRONE:-'v1.24'}"
+
+RUN echo "${K8S_VERSION}"
+
+RUN curl -sL https://github.com/rancher/k3s/releases/download/$(curl -Ls -o /dev/null -w %{url_effective} https://update.k3s.io/v1-release/channels/${K8S_VERSION} | awk -F/ '{ print $NF }')/${!K3S_BINARY} > /usr/local/bin/k3s && \
```

Comment:
It's good practice to enclose in double quotes strings that use interpolation in bash (to prevent word and glob splitting, if the expansion evaluates to values that contain spaces).